packages/legend-application-taxonomy/src/stores/StandaloneDataSpaceViewerStore.ts

```diff
@@ -111,24 +112,27 @@ export class StandaloneDataSpaceViewerStore {
         versionId,
       )) as Entity[];
 
-      // fetch dependencies
-      this.initState.setMessage(`Fetching dependencies...`);
-      const dependencyEntitiesIndex = (yield flowResult(
-        this.depotServerClient.getIndexedDependencyEntities(project, versionId),
-      )) as Map<string, Entity[]>;
-
       // analyze data space
-      this.initState.setMessage(`Analyzing data space...`);
       const analysisResult = (yield DSL_DataSpace_getGraphManagerExtension(
         this.graphManagerState.graphManager,
-      ).analyzeDataSpace(dataSpacePath, entities, dependencyEntitiesIndex, () =>
-        retrieveAnalyticsResultCache(
-          project.groupId,
-          project.artifactId,
-          versionId,
-          dataSpacePath,
-          this.depotServerClient,
-        ),
+      ).analyzeDataSpace(
```

Comment:
I think we should move all fetching logic (even fetching entities/project data) in right? We don't need any of them if we just call depot right?

Reply:
we should fetch projectData but i agree on the entities